templates/account/email/email_confirmation_message.txt

```diff
@@ -1,8 +1,7 @@
-{% load account %}{% user_display user as user_display %}{% load i18n %}{% autoescape off %}{% blocktrans with site_name=current_site.name site_domain=current_site.domain %}
-
-Welcome to Workbench!
-Please confirm your email address by clicking the link below. If you didn't sign up, you can safely ignore this email.
-
-{{ activate_url }}
-
-{% endblocktrans %}{% endautoescape %}
+{% load account %}
+{% user_display user as user_display %}
+{% load i18n_icu %}
+{% autoescape off %}
+  {% trans_html "py.templates.account.email.email_confirmation_message.confirmYourEmail" default="Welcome to Workbench! Please confirm your email address by clicking the link below. If you didn't sign up, you can safely ignore this email." %}
```

Comment:
These were two paragraphs. Please make them two translation strings, and make sure there's a newline between them.